static/app/views/replays/detail/domMutations/index.tsx

```diff
@@ -153,7 +150,7 @@ const StepConnector = styled('div')`
   position: absolute;
   height: 100%;
   top: 28px;
-  left: 31px;
+  left: 27px;
```

Comment:
This is still wonky

<img width="313" alt="Screen Shot 2022-08-18 at 1 43 02 PM" src="https://user-images.githubusercontent.com/187460/185491120-42f68728-8310-4574-a5c8-0ac8f1a76ea8.png">

how do breadcrumbs position the line?